src/App.js

```diff
@@ -2,15 +2,60 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
 
-  add100 = () => {};
+  increase = () => {
+    if (this.state.count % 5 === 0) {
```

Comment:
```suggestion
    if (this.state.count % 5 === 0 && this.state.count !== 0 ) {
```
Желательно добавить проверку для того, чтобы не было деления на 0